src/qt/bitcoingui.cpp

```diff
@@ -433,6 +440,24 @@ void BitcoinGUI::createActions()
             connect(activity, &CreateWalletActivity::created, this, &BitcoinGUI::setCurrentWallet);
             activity->create();
         });
+        connect(m_restore_wallet_action, &QAction::triggered, [this] {
+            QString backupFile = GUIUtil::getOpenFileName(this,
+                //: The title for Restore Wallet File Windows
+                tr("Load Wallet Backup"), QString(),
+                //: The file extension for Restore Wallet File Windows
+                tr("Wallet Data File (*.dat)"), nullptr);
+            if (backupFile.isEmpty()) return;
+
+            bool walletNameOk;
+            //: Title of the Restore Wallet input dialog (where the wallet name is entered)
+            //: Restore Wallet input dialog label
```

Comment:
The second line is a little confusing and feels redundant. I would suggest removing the second line.

Reply:
Done in b5ad1bd